samcli/local/common/runtime_template.py

```diff
@@ -123,6 +106,10 @@ def get_local_lambda_images_location(mapping, runtime):
     # nodejs runtimes in descending order
     "nodejs14.x",
     "nodejs12.x",
+    "nodejs10.x",
```

Comment:
Do we still need `nodejs10.x`? AFAIK, `nodejs10.x` has been deprecated. It's also removed in the app template repo.